### ISSUE TITLE
Using authenticated proxy

### ISSUE THREAD

**Issue by diegoortizmatajira** [adrienverge/openfortivpn#438]
I'm using openfortivpn on corporate network, where I have to use authenticated proxy
I use this script to initialize proxy settings

```
PROXY="http://user:password@proxy.company.com:8080"
export {http,https,ftp,rsync}_proxy=$PROXY
export {HTTP,HTTPS,FTP,RSYNC}_PROXY=$PROXY
```
Often I have to connect to VPN using forticlient VPN, but when it comes I have to use the VPN, I need to connect to a proxyless access point (my phone) and remove all proxy settings in order to use openfortivpn.

I've tried using the proxy sever IP instead of hostname (proxy.company.com) without success.

When I try to use openfortivpn with proxy settings the app doesn't show any text it just prints an empty line on the terminal.

Information about my configuration
OS: Manjaro Linux x86_64 
Kernel: 4.19.28-1-MANJARO 
Package: openfortivpn from AUR

**Comment by DimitriPapadopoulos** [comment]
As far as I know authentication is not implemented yet in proxy support code: ccaa359

**Comment by mrbaseman** [comment]
yes, authentication is one of the items on the agenda for improving the proxy support in #226

**Comment by Raoul555** [comment]
I vote +1 for this feature ;)

**Comment by jmartasek** [comment]
For reference I managed to overcome this by running a local squid proxy that is using the authenticated proxy as a parent. Sample  `squid.conf`:
```
http_port 3128
http_access allow all
cache_peer parent.proxy.example.com  parent   8080      0  no-query no-digest default login=username:password
never_direct allow all
```

**Comment by DimitriPapadopoulos** [comment]
We probably need [libproxy](https://github.com/libproxy/libproxy) for proper proxy support.
